Documentation/project-docs/clr-configuration-knobs.csx

```diff
@@ -0,0 +1,450 @@
+// This script generates documentation about the various configuration options that
+// are available and how to use them.
```

Comment:
Delete Documentation\project-docs>notepad clr-complus-conf-docgen.sh?

Reply:
Didn't want to do it before PR will be accepted